### PR TITLE
Add fixtures for public uploads to Demo

### PR DESCRIPTION
**Pull request by johnnyomair**
Showcase how public upload fixtures (e.g., for uploaded profile pictures) could be implemented in an application.

- [x] Add changeset (if necessary)

### CODE REVIEW

File: packages/api/cms-api/src/index.ts [hunk 1]
```diff
@@ -139,6 +139,7 @@ export { PageTreeNodeCategory, PageTreeNodeInterface, PageTreeNodeVisibility, Sc
 export { PageExists, PageExistsConstraint } from "./page-tree/validators/page-exists.validator";
 export { PublicUpload } from "./public-upload/entities/public-upload.entity";
 export { PublicUploadModule } from "./public-upload/public-upload.module";
+export { PublicUploadsService } from "./public-upload/public-uploads.service";
```
Comment (johnnyomair) [inline packages/api/cms-api/src/index.ts:142]: Do we need a changeset for this?

File: demo/api/src/db/fixtures/generators/public-uploads-fixture.service.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { download, PublicUpload, PublicUploadsService } from "@comet/cms-api";
```
Comment (nsams) [inline demo/api/src/db/fixtures/generators/public-uploads-fixture.service.ts:1]: `download` makes me :sob:

File: packages/api/cms-api/src/index.ts (hunk 1 above)
Comment (nsams) [reply packages/api/cms-api/src/index.ts:142]: it's a new api people can/should use in projects, so yes

Comment (johnnyomair) [reply packages/api/cms-api/src/index.ts:142]: 2e5c0886f5c49d2b2dc4c33a34e534a31f6a5c65

**Review by nsams — APPROVED** [review]

File: demo/api/src/db/fixtures/generators/public-uploads-fixture.service.ts (hunk 2 above)
Comment (johnnyomair) [reply demo/api/src/db/fixtures/generators/public-uploads-fixture.service.ts:1]: Let's continue the discussion here: https://github.com/vivid-planet/comet/pull/1521

**Pull request merged**